### ISSUE TITLE
Inconsistent grouping / display in booking form

### ISSUE THREAD

**Issue by shgysk8zer0** [SergeyMosin/Appointments#28]
Screenshots more useful than all the info given by Issues app. Don't think logs or anything would be helpful.

![2 self-destruct](https://user-images.githubusercontent.com/1627459/76434215-a1313680-6372-11ea-98cb-b3e51112fdec.png)
![1 self-destruct](https://user-images.githubusercontent.com/1627459/76434218-a2626380-6372-11ea-9260-0a8c0d6fb26c.png)

I'm seeing times for a day split between duplicate dates and a 5PM slot listed before an 11AM slot. All these are calendar default timezone, and is not something I saw for "floating."

Also saw when I accidentally deleted an "Available" slot and recreated in manually before timezone options existed (believe this was true even if set to "floating").

Sorry to appear to be complaining about your work. This is potentially very useful to me and I'm reporting the few oddities and hiccups that make this fall short of the potential. An immense "thank you" for creating this.

Wait... is the duplicate Fri. 13 an Easter Egg? That'd be kinda funny.

**Comment by SergeyMosin** [comment]
Is possible to see the .ics file for Wed. 11 5:00PM

**Comment by SergeyMosin** [comment]
Actually never mind, I think I know what the issue is. It is related to mixing different timezones.
Thanks

**Comment by shgysk8zer0** [comment]
That might be possible, but I just cleared out all previous appointments and recreated the schedule after the latest update. These should all be the same.

Oddly, I noticed that just dragging the 5PM to 10AM fixed everything (everything set for the week was accidentally set to 11-5 instead of 20-4). That happens to line up with my timezone offset, so I'm guessing appointments are grouped by UTC date rather than local date.
